src/usdc-balance-change/usdc-balance-change.spec.js

```diff
@@ -0,0 +1,185 @@
+        .mockResolvedValueOnce(ethers.BigNumber.from(5400000));
+
+      // get N balance updates (N=blockWindow)
+      await iterateHandlerExpectEmpty(data.blockWindow);
+
+      // the next block event will give us N+1 balances --> will trigger a balance check
+      const findings = await handleBlock();
+
+      // there should be no findings
+      expect(findings).toStrictEqual([]);
+
+      // do some additional checks on the balance history
+      const { balanceHistory } = data.addresses[TEST_ADDRESS];
+
+      // the current balance should match the last call to balanceOf()
+      expect(balanceHistory.slice(-1)[0]).toStrictEqual(new BigNumber(5400000));
+
+      // there should be 1 less balance recorded for each address at this point
+      // (the oldest was shifted out of the balance array for the next cycle)
+      expect(balanceHistory.length).toStrictEqual(data.blockWindow);
+    });
+
+    it('when the balance change is above the threshold but outside the block window', async () => {
+      // simulate a changing balance over the threshold, but too slow to trigger
+      // within the block window
+      // 6.0, 6.1, 6.2, 6.3, 6.4, 6.5, 6.6, 6.7 USDC
+      mockUsdcContract.balanceOf = jest.fn()
+        .mockResolvedValueOnce(ethers.BigNumber.from(6000000))
+        .mockResolvedValueOnce(ethers.BigNumber.from(6100000))
+        .mockResolvedValueOnce(ethers.BigNumber.from(6200000))
+        .mockResolvedValueOnce(ethers.BigNumber.from(6300000))
+        .mockResolvedValueOnce(ethers.BigNumber.from(6400000))
+        .mockResolvedValueOnce(ethers.BigNumber.from(6500000))
+        .mockResolvedValueOnce(ethers.BigNumber.from(6600000))
+        .mockResolvedValueOnce(ethers.BigNumber.from(6700000));
+
+      // get 8 balance updates
+      // there should be no findings
+      await iterateHandlerExpectEmpty(8);
+
+      // do some additional checks on the balance history
+      const { balanceHistory } = data.addresses[TEST_ADDRESS];
+
+      // the current balance should match the last call to balanceOf()
+      expect(balanceHistory.slice(-1)[0]).toStrictEqual(new BigNumber(6700000));
+
+      // there should be 1 less balance recorded for each address at this point
+      // (the oldest was shifted out of the balance array for the next cycle)
+      expect(balanceHistory.length).toStrictEqual(data.blockWindow);
+    });
+  });
+
+  describe('reports', () => {
+    it('when the balance change is above the threshold and within the block window', async () => {
+      // simulate a changing balance over the threshold
+      // 3.0, 3.1, 3.2, 3.3, 3.6 USDC
+      mockUsdcContract.balanceOf = jest.fn()
+        .mockResolvedValueOnce(ethers.BigNumber.from(3000000))
+        .mockResolvedValueOnce(ethers.BigNumber.from(3100000))
+        .mockResolvedValueOnce(ethers.BigNumber.from(3200000))
+        .mockResolvedValueOnce(ethers.BigNumber.from(3300000))
+        .mockResolvedValueOnce(ethers.BigNumber.from(3600000));
```

Comment:
Same comment as the previous two tests: Create a variable to store the `3600000` value and use it here and in the `expect` statement on line 178.

Reply:
Discussed offline; will leave as-is